sklearn/feature_selection/rfe.py

```diff
@@ -65,6 +67,30 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
         If within (0.0, 1.0), then ``step`` corresponds to the percentage
         (rounded down) of features to remove at each iteration.
 
+    tune_step_at : int or float or None, optional (default=None)
+        Number of remaining features reached when ``tuning_step`` is used
+        rather than ``step``. May be specified as an (integer) number of
+        remaining features or, if within (0.0, 1.0), a percentage (rounded
+        down) of the original number of features. If original number of
```

Comment:
I suspect this last sentence is too much technical detail.

Reply:
Should I just remove it?